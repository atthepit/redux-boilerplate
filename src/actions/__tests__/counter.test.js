import * as actions from '../counter'

describe('Counter action creators', () => {
  test('increment', () => {
    expect(
      actions.increment()
    ).toMatchSnapshot()
  })

  test('decrement', () => {
    expect(
      actions.decrement()
    ).toMatchSnapshot()
  })
})
