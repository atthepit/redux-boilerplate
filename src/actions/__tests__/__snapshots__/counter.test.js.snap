exports[`Counter action creators decrement 1`] = `
Object {
  "type": "DECREMENT",
}
`;

exports[`Counter action creators increment 1`] = `
Object {
  "type": "INCREMENT",
}
`;
